aptos-move/framework/aptos-framework/sources/reconfiguration.spec.move

```diff
@@ -1,10 +1,65 @@
 spec aptos_framework::reconfiguration {
     spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+
         // After genesis, `Configuration` exists.
         invariant [suspendable] chain_status::is_operating() ==> exists<Configuration>(@aptos_framework);
         invariant [suspendable] chain_status::is_operating() ==>
             (timestamp::spec_now_microseconds() >= last_reconfiguration_time());
     }
+    
+    /// A schem to ensure caller is admin
+    spec schema Init {
```

Comment:
Can you rename this to `AbortsIfNotAptosFramework`?

Reply:
The modification is complete.

Reply:
Done.